assets/javascripts/modules/checkout/promoCode.js

```diff
@@ -82,8 +81,8 @@ define([
     }
 
     function validate() {
-        var country = countryChoice.getCurrentCountryOption().value.trim(),
-            promoCode  = $inputBox.val().trim();
+        var country = formElements.BILLING.$COUNTRY_SELECT.val().trim(),
```

Comment:
So as discussed - for digipack any physical goods will go to your billing address, for paper we'll change this to use your delivery address country